tests/DummyTest.php

```diff
@@ -11,6 +11,6 @@ public function testIsHappy(): void
     {
         $dummy = new Dummy();
 
-        $this->assertTrue($dummy->isHappy());
+        $this->assertFalse($dummy->isHappy());
```

Comment:
Change to $this->assertTrue($dummy->isHappy());

Reply:
https://github.com/bsadnu/demo-circle/commit/e28167fac0fd99ba83c5c5dd70ae5085c13b117d